MAINTAINERS

```diff
@@ -1,2 +1,3 @@
+Ben Darnell <ben@bendarnell.com>
```

Comment:
We should add his github handler too. Ben Darnell <ben@bendarnell.com> (@bdarnell)

Also should we mention he is the maintainer of raft subpkg? Or people might cc him about other etcd issues.